CONTRIBUTING.rst

```diff
@@ -118,9 +118,9 @@ Documentation
 Documentation is generated using `Sphinx`_ and stored in the ``docs/`` directory.
 You can build it individually by running:
 
-.. code-block:: console
+.. prompt:: bash $
 
-   $ tox -e sphinx
+   tox -e sphinx
 
 You can view the result by opening ``docs/_build/html/index.html`` in your browser.
 
```

Comment:
all 'cd' commands show up in yellow,
is that intended ?

http://52.88.192.98:8080/job/ClusterHQ-flocker/job/remove-codeblocks-FLOC-1612/job/run_sphinx/ws/docs/_build/html/gettinginvolved/contributing.html